docs/source/sycamore/get_started/hardware.md

```diff
@@ -2,6 +2,6 @@
 
 We recommend at least 4GB of RAM to run Sycamore.
 
-We recommend using the Aryn Partitioner for the highest quality PDF processing. By default, the Aryn Partitioner is configured to use the Aryn Partitioning Service, a GPU-backed serverless endpoint. You can sign up for free [here](https://www.aryn.ai/get-started), and use it with your Sycamore job.
+We recommend using the Aryn Partitioner for the highest quality PDF processing. By default, the Aryn Partitioner is configured to use Aryn DocParse, a GPU-backed serverless endpoint. You can sign up for free [here](https://www.aryn.ai/get-started), and use it with your Sycamore job.
```

Comment:
let's not just say PDF here

Reply:
Leaving this as is since Aryn Partitioner locally can only do PDFs.